addons/website_event_track/static/src/js/event_track_reminder.js

```diff
@@ -30,55 +38,150 @@ publicWidget.registry.websiteEventTrackReminder = publicWidget.Widget.extend({
     _onReminderToggleClick: function (ev) {
         ev.stopPropagation();
         ev.preventDefault();
-        var self = this;
-        var $trackLink = $(ev.currentTarget).find('i');
+        var trackLink = ev.target;
+
+        this.opacityManagerElement = this.el.closest('.o_we_agenda_card') ?? this.el;
+        this.initialOpacity = this._getInitialOpacity();
 
         if (this.reminderOn === undefined) {
-            this.reminderOn = $trackLink.data('reminderOn');
+            this.reminderOn = trackLink.dataset.reminderOn;
         }
 
         var reminderOnValue = !this.reminderOn;
 
-        rpc('/event/track/toggle_reminder', {
-            track_id: $trackLink.data('trackId'),
-            set_reminder_on: reminderOnValue,
-        }).then(function (result) {
-            if (result.error && result.error === 'ignored') {
-                self.notification.add(_t('Talk already in your Favorites'), {
-                    type: 'info',
-                    title: _t('Error'),
-                });
-            } else {
-                self.reminderOn = reminderOnValue;
-                var reminderText = self.reminderOn ? _t('Favorite On') : _t('Set Favorite');
-                self.$('.o_wetrack_js_reminder_text').text(reminderText);
-                self._updateDisplay();
-                var message = self.reminderOn ? _t('Talk added to your Favorites') : _t('Talk removed from your Favorites');
-                self.notification.add(message, {
-                    type: 'info',
-                });
-                if (self.reminderOn) {
-                    Component.env.bus.trigger('open_notification_request', [
-                        'add_track_to_favorite',
-                        {
-                            title: _t('Allow push notifications?'),
-                            body: _t('You have to enable push notifications to get reminders for your favorite tracks.'),
-                            delay: 0
-                        },
-                    ]);
+        var trackId = parseInt(trackLink.dataset.trackId);
+
+        if (reminderOnValue){
+            //test toggle reminder
+            var self = this;
+            rpc('/event/track/toggle_reminder', {
+                track_id: trackId,
+                set_reminder_on: true,
+                save: false
```

Comment:
Ok I see now the `save` param. I'm not sure why we need to make a rpc call before doing potentially the same rpc call. From what I see, it should be possible to do this with a single rpc. Am I missing something here ?